kadai1/yukisato/conv/convertImages.go

```diff
@@ -0,0 +1,126 @@
+package conv
+
+import (
+	"errors"
+	"image/jpeg"
+	"image/png"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	ContentTypeJpeg = "image/jpeg"
```

Comment:
イニシャルはすべて小文字か大文字が推奨されています（このスタイルが採用されていることが多い気がします）
https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
	ContentTypeJPEG = "image/jpeg"
```

Reply:
ご確認、ご指摘ありがとうございます！
JPEGがキャピタルなのはうっかりしていました。
調べたらPNGも同様だったので合わせて修正し、 `ExtensionJpg` についても統一感がないので `extensionJPEG` としました（公開範囲も狭めました）
また `isPng` などのメソッドも同様に修正致しました。
ありがとうございました！